filebeat/harvester/harvester.go

```diff
@@ -48,11 +53,13 @@ func NewHarvester(
 	}
 
 	h := &Harvester{
+		Id:          uuid.NewV4(), // Unique identifier of each harvester
```

Comment:
It's not, see https://github.com/elastic/beats/pull/1144#discussion_r56020188 It is more I quite like UUID.